node/src/components/consensus/era_supervisor.rs

```diff
@@ -493,16 +493,16 @@ where
 
     pub(super) fn handle_linear_chain_block(
         &mut self,
-        block_header: BlockHeader,
+        block: Block,
```

Comment:
Why do we need a whole `Block` here if all you ever use are header's fields?

Reply:
I think it was because of `announce_block_handled` below.

Reply:
I thought it's needed by the `linear_chain` but it's for `linear_chain_sync`. Makes sense.